### PR TITLE
Fix plain redirect subjects

### PR DESCRIPTION
**Pull request by milescalabresi**
Fixes bug that causes emails to alias@site.learningu.org (plainredirects) to fail

Also updates some of the text to make it less ESP-specific

### CODE REVIEW

**Review by hwatheod — APPROVED** [review]
LGTM

**Comment by willgearty** [comment]
@milescalabresi did you already push this to all of the sites?

**Comment by milescalabresi** [comment]
No I was waiting for approval. I'll cherry pick it now

**Pull request merged**
